src/content/configuration/externals.md

```diff
@@ -76,7 +76,8 @@ externals: {
 
 ``` js
 externals : {
-  react: 'react'
+  react: 'React',
+  react-dom: 'ReactDOM'
```

Comment:
the property syntax is invalid - should be `'react-dom'`

Reply:
@chanoch Any update on this? :)